include/mesh/exodusII_io_helper.h

```diff
@@ -297,7 +297,8 @@ class ExodusII_IO_Helper : public ParallelObject
   /**
    * Sets up the nodal variables
    */
-  void initialize_element_variables(std::vector<std::string> names);
+  void initialize_element_variables(std::vector<std::string> names,
+                                    std::vector<std::set<subdomain_id_type>> vars_active_subdomains);
```

Comment:
IMHO this "helper" class isn't exactly a public API so we should be able to change it willy-nilly, but I'd like to hear others thoughts too.

In hindsight we should probably have enforced that by making the constructor private and making the user-friendly Exodus I/O class a friend...

Reply:
`vars_active_subdomains` should be passed in by const reference, if I understand correctly it is only read from.